spec/models/video_spec.rb

```diff
@@ -11,35 +12,47 @@
       result = Video.search_by_title("")
       expect(result).to eq([])
     end
-
     it "returns an array with one title when there is an exact match" do
-      Video.create(title: "black fish", description: "awesome")
+      Fabricate(:video, title: "black fish")
       result = Video.search_by_title("black fish")
       expect(result.length).to eq(1)
     end
-
     it "returns an array with more than one title when there is more than one match" do
-      Video.create(title: "black fish", description: "awesome")
-      Video.create(title: "black fish", description: "really awesome")
+      2.times { Fabricate(:video, title: "black fish") }
       result = Video.search_by_title("black fish")
       expect(result.length).to eq(2)
     end
-
     it "returns an array with two titles when there are two non-exact matches, ordered by newest first" do
-      black_fish = Video.create(title: "black fish", description: "awesome", created_at: 1.day.ago)
-      blackadder = Video.create(title: "blackadder", description: "really awesome")
+      black_fish = Fabricate(:video, title: "black fish", created_at: 1.day.ago)
+      blackadder = Fabricate(:video, title: "blackadder")
       result = Video.search_by_title("black")
       expect(result).to eq([blackadder, black_fish])
     end
-
     it "returns an empty array when there is an empty search string" do
-      Video.create(title: "black fish", description: "awesome")
-      Video.create(title: "black fish", description: "really awesome")
+      Fabricate(:video)
       result = Video.search_by_title("")
       expect(result).to eq([])
     end
-
   end
-  
 
+  describe "average_rating" do
```

Comment:
describe messages should be annotated with `#` or `.` depending on what method they are describing. - Read more [here](http://www.betterspecs.org/#describe)